src/Module/Settings/Display.php

```diff
@@ -224,10 +240,21 @@ protected function content(array $request = []): string
 			BBCode::PREVIEW_LARGE    => $this->t('Large Image'),
 		];
 
-		$network_timelines = $this->pConfig->get($uid, 'system', 'network_timelines', array_keys($this->getAvailableTimelines($uid, true)));
+		$bookmarked_timelines = $this->pConfig->get($uid, 'system', 'network_timelines', array_keys($this->getAvailableTimelines($uid, true)));
+		$enabled_timelines    = $this->pConfig->get($uid, 'system', 'enabled_timelines', array_keys($this->getAvailableTimelines($uid, false)));
+
 		$channel_languages = $this->pConfig->get($uid, 'channel', 'languages', [User::getLanguageCode($uid)]);
 		$languages         = $this->l10n->getAvailableLanguages(true);
-		$timelines         = $this->getAvailableTimelines($uid);
+
+		$timelines = [];
+		foreach ($this->getAvailableTimelines($uid) as $code => $timeline) {
```

Comment:
This should return Timeline entities now.

Reply:
This is changed.